imap_processing/lo/l0/lo_science.py

```diff
@@ -1,5 +1,6 @@
 """Processing function for Lo Science Data."""
```

Comment:
```suggestion
"""L0 processing functionality for Lo Science Data."""
```

Reply:
Agreed that it's a little confusing. My plan was to put all the Lo science packet processing in lo_science.py, but it might make sense to rename this. I'll think about this more and try to come up with something less confusing.